contracts/modules/LoanClosings/LoanClosingsWithGasToken.sol

```diff
@@ -1,119 +0,0 @@
-/**
```

Comment:
@tomcbean we need to call `replaceContract` with target ZERO to remove this contract. we need to do this so we won't forget